cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -587,10 +587,14 @@ func (r *TrafficOpsReq) CheckSystemServices() error {
 func (r *TrafficOpsReq) IsPackageInstalled(name string) bool {
 	for k, v := range r.Pkgs {
 		if strings.HasPrefix(k, name) {
+			log.Infof("Found in cache for '%v'", k)
```

Comment:
nit: formatting parameter could be more specific; strings may use `%s` for better compile-time safety

Reply:
changed to `%s`